fdbclient/CoordinationInterface.h

```diff
@@ -236,4 +239,30 @@ struct ProtocolInfoRequest {
 	}
 };
 
+// Returns true if the cluster descriptor may be modified.
+struct CheckDescriptorMutableReply {
+	constexpr static FileIdentifier file_identifier = 7784299;
+	CheckDescriptorMutableReply() = default;
+	explicit CheckDescriptorMutableReply(bool value) : value(value) {}
+	bool value;
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, value);
+	}
+};
+
+// Allows client to check if allowed to change the cluster descriptor.
+struct CheckDescriptorMutable {
```

Comment:
Typically, these messages are suffixed with the work `Request`:

```suggestion
struct CheckDescriptorMutableRequest {
```